app/controllers/orders_controller.rb

```diff
@@ -7,4 +7,18 @@ def show
     @order = Order.find(params[:id])
   end
 
+  def create
+    user = User.find(session[:user_id].to_i)
+
+    order = user.orders.create(total: @cart.total_price.to_s)
+
+    @cart.contents.each do |id, quantity|
+      order.order_creatures.create(creature: Creature.find(id),
```

Comment:
@bethsebian @Carmer  Group question: we are manually creating order_creatures (a join table here) and are wondering if there is an alternative way to dynamically populate the quantity column which lives on the join table order_creatures? Or is the way we are doing it a good/acceptable strategy?